src/components/common/SearchFiltersWrapper/index.tsx

```diff
@@ -0,0 +1,130 @@
+import React, { useMemo } from 'react';
+import { Box, Grid } from '@mui/material';
+import { PillList, Textfield } from '@terraware/web-components';
+import { Option } from '@terraware/web-components/components/table/types';
+import strings from 'src/strings';
+import { FieldOptionsMap, FieldValuesPayload, SearchNodePayload } from 'src/types/Search';
+import { FilterField } from 'src/components/common/FilterGroup';
+import useDeviceInfo from 'src/utils/useDeviceInfo';
+import IconFilters from './IconFilters';
+import FeaturedFilters from './FeaturedFilters';
+
+export type SearchInputProps = {
+  search: string;
+  onSearch: (search: string) => void;
+};
+
+// A "featured filter" is one that resides within its own multi select, between the search bar and the filters hidden
+// behind the filter icon (those are now called "icon filters"). Since the implementer should not care about where the
+// filters are (featured or icon), the `filtersProps.setFilters` and `filterProps.filters` passed in are really used to
+// control "all filters". Ideally the SearchFiltersProps gets turned into IconFiltersProps, and those two aforementioned
+// properties get put at the top level of the SearchProps, to make it more obvious. That is a slightly bigger refactor
+// than I want to embark on right now since there are a good number of places using this component. One side effect of
+// this is that **you can't use featured filters without the icon filters also being set up.** As soon as we decide to
+// implement this wrapper with only the featured filters, it makes sense to do the refactor. Hopefully this note
+// clears up any confusion and serves as a reminder that we should do that in the future.
```

Comment:
Illustrate what the props might look like, just to help further understand.

Reply:
Let me know what you think!

Reply:
Thanks @nickgraz this is super helpful 🙏 and I get the picture now!!